distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-framework-camel-component.adoc

```diff
@@ -27,9 +24,8 @@ catalog:framework
 
 ===== Sending Messages to Catalog Framework Endpoint
 
-====== Catalog Framework Producer
-
-In Producer mode, the component provides the ability to provide different inputs and have the Catalog framework perform different operations based upon the header values.  
+.Catalog Framework Producer
+In Producer mode, the component provides the ability to supply different inputs and have the Catalog framework perform different operations based upon the header values.  
```

Comment:
❔ Not sure if it matters but are we consistent on the capitalization of `framework`? in Catalog framework?

Reply:
good catch. It should be consistent